cmd/amp/commands.go

```diff
@@ -128,7 +129,7 @@ func addCommands(cmd *cobra.Command, c cli.Interface) {
 
 func info(c cli.Interface) error {
 	s := c.Server()
-	c.Console().Infof("[%s]\n", s)
+	fmt.Fprintf(c.Err(), "[%s]\n", s)
```

Comment:
the `info()` func displays the context information, but I changed the output from std out to std err to make it work for the rm commands with multiple arguments. Maybe the func can be renamed to `displayContextInfo()`